### DISCUSSION TITLE
ReadMe Base

### DISCUSSION THREAD

**Discussion by LoriaLawrenceZ** [ZynkePartnership/SIASO#18] [General]
@Paiva8859 , mandei aqui o repositório que estou me baseando.
Caso você venha a mexer no reademe, utiliza esse modelo apenas como base, copia apenas o esqueleto, a ESTRUTURA do modelo, mas o conteúdo deve ser totalmente original e que apresente o conteúdo de forma direta e objetiva 👍 

Repositório:
https://github.com/shyoutarou/README-Model

**Comment by TheuZCoder** [comment]
brabo
